src/agent/commands/queries.js

```diff
@@ -17,6 +17,13 @@ export const queryList = [
             let pos = bot.entity.position;
             // display position to 2 decimal places
             res += `\n- Position: x: ${pos.x.toFixed(2)}, y: ${pos.y.toFixed(2)}, z: ${pos.z.toFixed(2)}`;
+            // Environmental Awareness
+            res += `\n- Block Above: ${world.getBlockAtPosition(bot, 0, 2, 0).name}`;
```

Comment:
can you share the duplicate functionality into a function that returns the text describing block at head, below, etc